src/OSPSuite.Core/Domain/RunOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace OSPSuite.Core.Domain
+{
+   public class RunOptions
+   {
+      /// <summary>
+      ///    (Maximal) number of cores to be used (1 per default)
+      /// </summary>
+      public int NumberOfCoresToUse { get; set; } = 1;
```

Comment:
This is the most basic option now that needs to be passed along to all our runner. We did not have that for sensitivity